src/App.js

```diff
@@ -50,30 +108,31 @@ function App() {
     return (
       <div className="App">
         <header>
-          <h1>Check out these movies</h1>
+          <h1>Moviesearch</h1>
 
-          {/* This bit should be its own searchbar component */}
-          <form onSubmit={handleSubmit}>
-            <label htmlFor="art">Search for movies:</label>
-            <input type="text" id="search" onChange={handleInput} value={userInput}/>
-            <button>Search</button>
-          </form>
+          <div className="formContainer">
+            <form onSubmit={handleSubmit}>
+              <label htmlFor="movie">Title</label>
+              <input type="text" id="search" onChange={handleInput} value={userInput} placeholder='Enter Movie Title'/>
+              <button>Search</button>
+            </form>
 
+            <form onSubmit={handleSubmit2}>
+              <label htmlFor="crew">Crew member</label>
+              <input type="text" id="search" onChange={handleInput2} value={userInput2} placeholder='Enter Person' />
+              <button>Search</button>
+            </form>
+          </div>
         </header>
 
-      <section className="results">
-        {movies.map((movie) => {
-          return (
-            <div key={movie.id} className="movieCard">
-              <h2>{movie.original_title}</h2>
-              {movie.poster_path === null
-              ? <p>No poster to display.</p>
-              : <img src={`https://image.tmdb.org/t/p/original/${movie.poster_path}`} alt={movie.original_title} />}
-            </div>
-            )
-          })
-        }
-      </section>
+        {alert === false
```

Comment:
Never even thought of this 🤯 This is a great way to set the error handling as a condition